web_application/src/CreateProfilePge.js

```diff
@@ -0,0 +1,274 @@
+import React, { useState } from "react";
+//import { useNavigate } from 'react-router-dom';
+import "./styles.css";
+
+function CreateProfilePage({}) {
+    //const Navigate = useNavigate();
+    const [fullName, setFullName] = useState('');
+    const [gender, setGender] = useState('');
+    const [dateOfBirth, setBirthDate] = useState('');
+    const [smoker, setSmoker] = useState(false);
```

Comment:
useState(false) is a potential bug as if user didn't choose gender and save the data we will see it has false. maybe should do, useState('')